src/canvas.class.js

```diff
@@ -712,30 +709,25 @@
       ctx.lineWidth = this.selectionLineWidth;
       ctx.strokeStyle = this.selectionBorderColor;
 
+      minX += strokeOffset;
+      minY += strokeOffset;
+      maxX -= strokeOffset;
+      maxY -= strokeOffset;
       // selection border
       if (this.selectionDashArray.length > 1 && !supportLineDash) {
-
-        var px = groupSelector.ex + STROKE_OFFSET - ((left > 0) ? 0 : aleft),
-            py = groupSelector.ey + STROKE_OFFSET - ((top > 0) ? 0 : atop);
-
         ctx.beginPath();
 
-        fabric.util.drawDashedLine(ctx, px, py, px + aleft, py, this.selectionDashArray);
-        fabric.util.drawDashedLine(ctx, px, py + atop - 1, px + aleft, py + atop - 1, this.selectionDashArray);
-        fabric.util.drawDashedLine(ctx, px, py, px, py + atop, this.selectionDashArray);
-        fabric.util.drawDashedLine(ctx, px + aleft - 1, py, px + aleft - 1, py + atop, this.selectionDashArray);
+        fabric.util.drawDashedLine(ctx, minX, minY, maxX, minY, this.selectionDashArray);
+        fabric.util.drawDashedLine(ctx, minX, maxY, maxX, maxY, this.selectionDashArray);
+        fabric.util.drawDashedLine(ctx, minX, minY, minX, maxY, this.selectionDashArray);
+        fabric.util.drawDashedLine(ctx, maxX, minY, maxX, maxY, this.selectionDashArray);
```

Comment:
would you check if IE11 supports dashed stroke? if it does, we can finally get rid of  the block (!supportedLineDash)

Reply:
Here is the support link. Looks like IE 11 supports but I'm not sure if the support is good enough on other browsers like chrome? https://caniuse.com/mdn-api_canvasrenderingcontext2d_setlinedash

Reply:
yes setLineDash is an old feature that was missing on ie10 only.
Since we do not support ie10 anymore, this is safe to go.